src/main/java/core/basesyntax/strategy/actions/SupplyActionHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.actions;
+
+import java.util.Map;
+
+public class SupplyActionHandler implements ActionHandler {
+    @Override
+    public void apply(Map<String, Integer> map, String data) {
+        String key = data.split(SPLITTER, 2)[0];
+        Integer value = Integer.parseInt(data.split(SPLITTER, 2)[1]) + map.get(key);
```

Comment:
data.split(SPLITER,2) - put in local variable and use further where needed
make constants fo magic number 0,1 and 2
NumberFormatException - handle for parseInt